README.md

```diff
@@ -10,5 +10,9 @@
 
   <p>Have fun and good luck!</p>
 
-  <p>P.S. Want to contribute? Feel free to send a pull request to this course’s
-  <a href=”https://github.com/jetbrains-academy/introduction_to_python”>git repo</a>.</p>
+## Want to know more?
+If you have questions about the course or the tasks or if you find some errors,
+you can ask questions and participate in discussions in repository [issues](https://github.com/jetbrains-academy/introduction_to_python/issues).
```

Comment:
feel free to ask questions and participate in discussions within the repository [issues]